src/MovieDetails/MovieDetails.js

```diff
@@ -12,20 +11,18 @@ class MovieDetails extends Component {
     }
   }
 
-  hideDetails = () => {
-    this.setState({showDetails: false})
-  }
-
   render() {
     return (
-      <div className="details" style={{ backgroundImage: `url(${this.props.movie.backdrop_path})`, opacity: .5 }}>
+      <div className="details" style={{ backgroundImage: `linear-gradient(to top, black, 60%, transparent), url(${this.props.movie.backdrop_path})`}}>
```

Comment:
I would remove all the `style`s from your components and instead put ALL styling in your CSS file. Having styling live sometimes in CSS and sometimes in the component itself is inconsistent.